### ISSUE TITLE
Flake8Command.run() raises SystemExit even on success.

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8#82]
In GitLab by @mforbes on Mar 8, 2015, 14:22

Even when successful, `` Flake8Command.run()`` will raise a ``SystemExit(0)`` exception, making it difficult to work with (my use-case was writing a custom test command that runs flake8, nose tests, etc.)  Please consider ananlogous change to 236e0f0ea1d74a5acaef074d7494184fa9f1fa1b: 

```
    if exit_code > 0:
        raise SystemExit(exit_code > 0)
```

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Mar 8, 2015, 20:56

Feel free to send a merge request.

**Comment by asottile** [comment]
In GitLab by @mforbes on Mar 8, 2015, 21:17

mentioned in merge request !23

**Comment by asottile** [comment]
In GitLab by @mforbes on Mar 10, 2015, 20:00

mentioned in commit da2b0b8a43d7765b4e64e43365406d9678791975

**Comment by asottile** [comment]
In GitLab by @mforbes on Mar 10, 2015, 20:00

Status changed to closed by commit 1fd8b661749f90b51299fbc733a8e7fe7021968b
